libvast/src/system/connector.cpp

```diff
@@ -140,12 +140,14 @@ std::string format_time(caf::timespan timespan) {
 void log_connection_failed(connect_request request,
                            caf::timespan remaining_time,
                            caf::timespan retry_delay) {
-  VAST_INFO("client faild to connect to remote node {} ({}):{}; attempting to "
+  const auto resolved_host
+    = caf::io::network::interfaces::native_address(request.host)->first;
+  VAST_INFO("client faild to connect to remote node {}:{}{}; attempting to "
             "reconnect in {} (remaining time: {})",
-            request.host,
-            caf::io::network::interfaces::native_address(request.host)->first,
-            request.port, format_time(retry_delay),
-            format_time(remaining_time));
+            request.host, request.port,
+            resolved_host != request.host ? fmt::format(" ({})", resolved_host)
```

Comment:
Small nit: we should probably `tolower()` the `request.host`, just in case an IPv6 address with `A-F` is submitted.

Reply:
Implemented in 73d31a1073